### ISSUE TITLE
Unresolved reference: `Result` 

### ISSUE THREAD

**Issue by spongecaptain** [intellij-rust/intellij-rust#9922]
## Environment

* **IntelliJ Rust plugin version:** 0.4.171.4656-213
* **Rust toolchain version:** 1.64.0 (a55dd71d5 2022-09-19) x86_64-apple-darwin
* **IDE name and version:** CLion 2021.3.4 (CL-213.7172.20)
* **Operating system:** macOS 12.5.1
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description

Unresolved reference: `Result`  error, seen below.

<img width="512" alt="image" src="https://user-images.githubusercontent.com/48505670/209750184-8a67e61e-4364-4248-ad13-580c71170b86.png">
<img width="812" alt="image" src="https://user-images.githubusercontent.com/48505670/209750206-2ee441e5-9dfa-41af-81bd-1aacaa7d0a3e.png">

## Steps to reproduce

Just use the `Result` type in ide, such problem will be reproduced.

**Comment by mili-l** [comment]
Hello @spongecaptain , could you [update](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#update) Rust plugin:
`Settings / Preferences | Plugins`, search for Rust and click `Update`
If the problem persists, feel free to let us know

**Comment by spongecaptain** [comment]
> Hello @spongecaptain , could you [update](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#update) Rust plugin: `Settings / Preferences | Plugins`, search for Rust and click `Update` If the problem persists, feel free to let us know

Hi @mili-l, when I right clicked the Rust plugin, I found there are only two buttons: Disable and Uninstall. I did not find the way to update it. Is it possible that `0.4.171.4656-213` is the latest version?

<img width="1572" alt="image" src="https://user-images.githubusercontent.com/48505670/209894322-c19fbb0d-1fdc-4247-a258-b09a2511af9e.png">

**Comment by rust1248** [comment]
No, the latest version is 0.4.185.5086, but I don't think you can install it because it's only available to CLion 2022.2 and after

**Comment by mili-l** [comment]
yeah, indeed, latest Rust plugin version compatible starting from 2022.2
However, @spongecaptain, could you upgrade your Clion?

**Comment by spongecaptain** [comment]
After I update Clion to latest version, such problem still exist.
<img width="663" alt="image" src="https://user-images.githubusercontent.com/48505670/209962778-af89823c-0c96-4c69-ad2d-8b8ecf06fa15.png">
And the Rust plugin is `0.4.185.5086-223`, see below:
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/48505670/209962857-9b285295-406a-49ec-8ae9-027350f76925.png">

The problem is here

<img width="875" alt="image" src="https://user-images.githubusercontent.com/48505670/209963000-7963f999-142d-42f1-9a4a-1505ca2690cf.png">

**Comment by spongecaptain** [comment]
I dont know why. But After using the command `rustup update` to update rust version `1.64.0` to rust `1.66.0`, I found it work well now.

I suspect the `rust-analyer` is to blame, the version is not compatible, but not sure about it.
